docs/ciphers.rst

```diff
@@ -80,30 +78,29 @@ art. However, the Let's Encrypt certificate authority does *not*
 dictate end-users' security policy, and any site is welcome to change
 its preferences in accordance with its own policy or its administrators'
 preferences, and use different cryptographic mechanisms or parameters,
-or a different priority order, than the defaults provided by the Let's
-Encrypt client.
+or a different priority order, than the defaults provided by Certbot.
 
-If you don't use the Let's Encrypt client to configure your server
-directly, because the client doesn't integrate with your server software
-or because you chose not to use this integration, then the cryptographic
-defaults haven't been modified, and the cryptography chosen by the server
-will still be whatever the default for your software was.  For example,
-if you obtain a certificate using *standalone* mode and then manually
-install it in an IMAP or LDAP server, your cryptographic settings will
-not be modified by the client in any way.
+If you don't use Certbot to configure your server directly, because the 
```

Comment:
I would recommend against reflowing text, unless there are really egregious line length issues, since it makes `git blame` less useful.